setup.py

```diff
@@ -247,7 +247,7 @@
         ),
         "google-auth >= 2.14.1, <3.0.0dev",
         "proto-plus >= 1.22.0, <2.0.0dev",
-        "protobuf>=3.19.5,<5.0.0dev,!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5",
+        "protobuf>=3.20.2,<6.0.0dev,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5",
```

Comment:
It's not clear if the minimum version should be `3.20.2` which matches [gapic-generator-python](https://github.com/googleapis/gapic-generator-python/blob/6fc3d22b697ed14ef2daa18bbd0964026caed850/gapic/templates/setup.py.j2#L39) or `4.25.3` which is in `main` 

https://github.com/googleapis/python-aiplatform/blob/80fa75c50bd65e6ead39568ee8ceb12f70733fa6/setup.py#L250